src/exceptions/HttpException.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Base class for creating exceptions.
+ */
+export default class HttpException extends Error {
```

Comment:
Can we use something that exists as opposed to rolling our own? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/http-errors/index.d.ts